src/Morphir/SpringBoot/Backend.elm

```diff
@@ -208,19 +207,20 @@ mapStatefulAppImplementation opt distribution currentPackagePath currentModulePa
 
                             Just (CustomTypeSpecification _ constructors) ->
                                 constructors
-                                    |> Dict.toList
                                     |> List.concatMap
-                                        (\( _, types ) ->
-                                            types
-                                                |> List.concatMap
-                                                    (\( _, consType ) ->
-                                                        case consType of
-                                                            Type.Reference _ (FQName _ _ consTypeName) _ ->
-                                                                [ consTypeName |> Name.toTitleCase ]
-
-                                                            _ ->
-                                                                []
-                                                    )
+                                        (\constructor ->
+                                            case constructor of
+                                                Type.Constructor _ types ->
+                                                    types
+                                                        |> List.concatMap
+                                                            (\( _, consType ) ->
+                                                                case consType of
+                                                                    Type.Reference _ ( _, _, consTypeName ) _ ->
+                                                                        [ consTypeName |> Name.toTitleCase ]
+
+                                                                    _ ->
+                                                                        []
+                                                            )
                                         )
```

Comment:
Same merge issue here. You should keep my code without the `Type.Constructor` which I removed.